### PR TITLE
Return Collection of results rather than just the first one.

### PR DESCRIPTION
**Pull request by danswiser**
I started with this project when I was looking to create an address search drop-down. Rather than selecting the first result, which I found to rarely be the one I needed, I wanted to have a collection of results instead. I've re-written this so that it doesn't break changes with the original `getCoordinatesForAddress()`.

This pr adds `getAllCoordinatesForAddress()` which will return an `Illuminate\Support\Collection` object containing all of the results.

### CODE REVIEW

**Comment by freekmurze** [comment]
Could you add a test for `getAllCoordinatesForAddress`?

**Comment by Pes8** [comment]
need this!

File: src/Geocoder.php [hunk 1]
```diff
@@ -102,37 +128,37 @@ public function getAddressForCoordinates(float $lat, float $lng): array
...
-            return $this->emptyResponse();
+            return $this->emptyResponse()->first();
         }
-
-        return $this->formatResponse($reverseGeocodingResponse);
+        return $this->formatResponse($reverseGeocodingResponse)->first();
     }
 
-    protected function formatResponse($response): array
+    protected function formatResponse($response): Collection
```
Comment (freekmurze) [inline src/Geocoder.php]: Could you refactor this so a collection is not used? Just use a regular array instead.

**Comment by Pes8** [comment]
Any news ? :(

File: src/Geocoder.php (hunk 1 above)
Comment (Pes8) [reply src/Geocoder.php]: @freekmurze I've opened another PR #73 with your request

Comment (danswiser) [reply src/Geocoder.php]: Thanks @Pes8.

PM decided not to use the project so I didn't come back to this right away.

**Pull request closed without merging**